brianlong/www.validators.app
# 179475514 sync epoch index with epoch credits

#### What's this PR do?
validators api endpoint returns epoch in which validator was last updated

#### How should this be manually tested?
 - run gather_rpc daemon
 - see the validators endpoint (/api/v1/validators/mainnet) should include epoch of lastly created validator_history for each validator

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/179475514

#### Task completed checklist:
- [Y] Is there appropriate test coverage?
- [Y] Did I take into consideration possible security vulnerabilities?
- [Y] Are the controllers secure?
- [Y] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
